tests/solutions/test_models.py

```diff
@@ -1,101 +1,61 @@
 # flake8: noqa
 # pylint: disable=all
```

Comment:
These linters should not be disabled anymore, now that you have cleaned up this file (just remove the two comments). Do a `make lint` afterwards to find and fix any style issues.